Lib/subprocess.py

```diff
@@ -712,7 +712,12 @@ def __init__(self, args, bufsize=-1, executable=None,
                  restore_signals=True, start_new_session=False,
                  pass_fds=(), *, encoding=None, errors=None, text=None):
         """Create new Popen instance."""
-        _cleanup()
+        if not _mswindows:
+            # Unlike Unix, a process in Windows doesn't have to be waited on
+            # by its parent to avoid a zombie. Keeping the handle open will
+            # actually create a zombie until the next _cleanup() call, which
+            # may be never if Popen() isn't called again.
+            _cleanup()
```

Comment:
Can't you modify _cleanup() to do nothing if _active is None, rather than modify the code here?

Just initialize _active to None on Windows, to make Popen code look the same on all platforms.

Reply:
Great point! Adjusted my patch. Thanks!